### DISCUSSION TITLE
What if someone takes my email on GitHub? How can I recover it?

### DISCUSSION THREAD

**Discussion by alexcmgit** [community/community#23521] [New to GitHub]
<p>If someone link my email as a secondary email in their GitHub account, I’ll not be able to create any account linked to my email anymore, since GitHub will say that the email is already in use (This happens even if the email is not verified by the user who took my email)</p>
<p>How can I proceed? How can I claim my email to my account?</p>
<h3>
<a name="tldr-1" class="anchor" href="#tldr-1"></a>TL;DR</h3>
<p>Let two users, <strong>Abel</strong> and <strong>John</strong>. <strong>Abel</strong> already have a GitHub account and a verified email <strong><a href="mailto:abel@email.com">abel@email.com</a></strong>, <strong>John</strong> doesn’t.</p>
<p><strong>Abel</strong> knows the email of <strong>John</strong> (<strong><a href="mailto:john@email.com">john@email.com</a></strong>) and then proceed to link his email as a secondary email, now <strong>Abel</strong> has two linked emails into your GitHub account (<strong>abel@email</strong> as primary and verified email, and <strong><a href="mailto:john@email.com">john@email.com</a></strong> as secondary but not verified)</p>
<p>Now, if <strong>John</strong> tries to create a new GitHub account using your own email <strong><a href="mailto:john@email.com">john@email.com</a></strong> GitHub doesn’t allow and shows a message <code>Email invalid or already in use</code></p>
<p><strong>Abel</strong> can’t verify the email since the email is owned by <strong>John</strong>. So, how can <strong>John</strong> proceed and claim his email?</p>
<p><strong>Image exempliflies the Abel GitHub email configuration:</strong></p>
<p><a href="https://i.stack.imgur.com/XbSfO.png" rel="noopener nofollow ugc"><img src="https://i.stack.imgur.com/XbSfO.png" alt="enter image description here" width="612" height="500"></a></p>
<p><strong>The image below exempliflies John view:</strong></p>
<p><a href="https://i.stack.imgur.com/y7whl.png" rel="noopener nofollow ugc"><img src="https://i.stack.imgur.com/y7whl.png" alt="enter image description here" width="680" height="304"></a></p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>yup, that’s a possibility, particularly if an old email with commits or even so, badges,</p>
<p>so you want to make sure, you take it before anyone else</p>
<p>if in your case, contact support, hopefully they will respond,</p>
<p><a href="https://support.github.com/contact/account">https://support.github.com/contact/account</a></p>
<p>if you know the other person using your email, you can report him too by going to his / her profile and clicking report, lower left of the profile</p>
<p>this is almost the same problem with usernames that are not released,</p>

**Reply by adamc00** [reply]
[Unlinking your email address from a locked account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account) should probably be attempted before contacting support.

**Comment by mrthawzinoo** [comment]
How to recover unlinking account ?
